src/main/webapp/app/shared/model/firebase/firebase.model.ts

```diff
@@ -382,12 +383,13 @@ export class Comment {
 export class Review {
   updateTime: number;
   updatedBy = '';
-  lastReviewed?: string;
-  // These two properties should not coexist
+  lastReviewed?: string | Object;
```

Comment:
I assume you disabled  @typescript-eslint/ban-types  because of this? It seems `object` is recommended. https://typescript-eslint.io/rules/ban-types/

Reply:
I updated to what the types should actually be: `string | CancerType[]`. The `object` type was also against the rules.